doc/rst/opt_adat_rx.rst

```diff
@@ -0,0 +1,72 @@
+|newpage|
+
+ADAT Receive
+============
+
+The codebase supports a single ADAT receiver that can receive up to eight channels of audio at a sample rate
+of 44.1kHz or 48kHz over an optical interface.
+In order to provide ADAT functionality ``lib_xua`` uses `lib_adat <https://www.github.com/xmos/lib_spdif>`_.
+
+Basic configuration of ADAT receive functionality is achieved with the following :ref:`defines<opt_adat_rx_defines>`:
+
+.. _opt_adat_rx_defines:
+
+|beginfullwidth|
+
+.. list-table:: ADAT RX defines
+   :header-rows: 1
+   :widths: 25 40 40
+
+   * - Define
+     - Description
+     - Default
+   * - ``XUA_ADAT_RX_EN``
+     - Enable ADAT receive
+     - ``0`` (Disabled)
+   * - ``ADAT_RX_MAX_CHANS``
+     - Maximum number of channels to receive over the ADAT interface
+     - ``8, 4 or 2`` depending on the min and max sampling freq supported in the application
+   * - ``ADAT_RX_INDEX``
+     - Start channel index of ADAT RX channels
+     - N/A (must be defined by the application)
+
+|endfullwidth|
+
+.. note::
+
+   ADAT receive always runs on the tile defined by ``AUDIO_IO_TILE``
+
+The codebase expects the ADAT receive port to be defined in the application XN file as ``PORT_ADAT_IN``.
+This must be a 1-bit port, for example::
+
+    <Port Location="XS1_PORT_1O"  Name="PORT_ADAT_IN"/>
+
+When ADAT receive is enabled the codebase expects to either drive a synchronisation signal to an external
+Cirrus Logic CS2100 device or use lib_sw_pll (xcore.ai only) for generating a master clock that is synchronised
```

Comment:
Add link to https://www.xmos.com/file/lib_sw_pll

Reply:
done